Dockerfile

```diff
@@ -1,42 +1,29 @@
-# Set the base image to Python
-FROM python:2.7.9
-MAINTAINER EclecticIQ <opentaxii@eclecticiq.com>
+FROM python:3.6-stretch AS build
+LABEL maintainer="EclecticIQ <opentaxii@eclecticiq.com>"
 
-# Create the working dir and set the working directory
-WORKDIR /
+RUN python3 -m venv /venv && /venv/bin/pip install -U pip setuptools
```

Comment:
You don't pin package versions, okay :). You know my opinion, but I don't insist.

Reply:
For `pip` and `setuptools` in particular, I'm of the opinion that our packages should *always* work with the latest versions, so I'm okay with not version-pinning them.